src/main/java/com/rocketden/main/service/SubmitService.java

```diff
@@ -58,4 +105,58 @@ public SubmissionDto submitSolution(Game game, SubmissionRequest request) {
 
         return GameMapper.submissionToDto(submission);
     }
+
+    // Sends a POST request to the tester service to judge the user submission
+    protected Submission callTesterService(TesterRequest request) {
+        try {
+            HttpPost post = new HttpPost(getTesterUrl());
+
+            StringEntity stringEntity = new StringEntity(gson.toJson(request));
+            post.setEntity(stringEntity);
+            post.setHeader("Content-type", "application/json");
+
+            HttpResponse response = httpClient.execute(post);
+            String jsonResponse = EntityUtils.toString(response.getEntity());
+
+            TesterResponse testerResponse = getResponseFromJson(jsonResponse);
+
+            // TODO: logic to convert TesterResponse into Submission object
+            Submission submission = new Submission();
+            submission.setNumCorrect(request.getProblem().getTestCases().size());
+            submission.setNumTestCases(request.getProblem().getTestCases().size());
+            submission.setPlayerCode(new PlayerCode(request.getCode(), request.getLanguage()));
```

Comment:
So to be clear, right now, we are getting the results from the tester service but not doing anything with them because that process needs more work?

Reply:
Yeah - I think kind of tied to your other comment, once the tester service returns a more complete TesterResponse, then we would actually be able to use it